source/docs/magento1/exposing-additional-attributes.md

```diff
@@ -0,0 +1,106 @@
+---
+id: exposing-additional-attributes
+title: Exposing additional attributes
+---
+
+In case where you need to expose new attributes on the existing endpoint you can simply add this attribute on your `api2.xml` file. For that, you need to retrieve resource name (API_UNIQUE_NAME value if you follow the basic structure shown in "Add custom endpoint section") and add your attribute(s) into the `attributes` node.
```

Comment:
Before starting the documentation page, we should explain why this can be useful. A message like:

> In many shops, there are more attributes than the standard ones available in Magento. This can be done following this [Magento documentation] but it then needs to be fetched by Front-Commerce. This can be done by exposing the newly created attributes in the endpoints already used in Front-Commerce. For instance, when retrieving products, Front-Commerce uses the `/api/rest/products` URL.
>
> In this documentation guide, you will learn how to add these attributes to any endpoint.


Then you can introduce the api2.xml, and continue with your documentation.

Reply:
Even though there is another page explaining where to put the `api2.xml`, we should repeat it here.